plugins/dell/fu-dell-smi.c

```diff
@@ -169,26 +163,26 @@ fu_dell_query_dock (FuDellSmiObj *smi_obj, DOCK_UNION *buf)
 	result = fu_dell_get_res (smi_obj, cbARG1);
 	if (result != SMI_SUCCESS) {
 		if (result == SMI_INVALID_BUFFER) {
-			g_debug ("Invalid buffer size, needed %" G_GUINT32_FORMAT,
-				 fu_dell_get_res (smi_obj, cbARG2));
+			g_debug (
+			    "Invalid buffer size, needed %" G_GUINT32_FORMAT,
```

Comment:
I'm not really a fan of this indent; can we force clang to always put the first part next to the bracket?

Reply:
I can't seem to find a way to correct this, it happens on long lines.  @YehezkelShB anythoughts?

Reply:
It can be `AlignAfterOpenBracket`, the effect we see here looks like `AlwaysBreak` option while the project currently uses `Align` style.

Reply:
The peculiar thing is the clang format is currently set as:
`AlignAfterOpenBracket: Align`

But that still happens for a few cases.  It might just be because that string is considered so long before the split of the comma.

Reply:
By using penalties I've gotten it to do closer to what you expect (basically splitting the longer string).